### ISSUE TITLE
A "basic team site" does not exist as indicated by documentation

### ISSUE THREAD

**Issue by thernstig** [MicrosoftDocs/OfficeDocs-O365ITPro#170]
Text in this documentation indicates that Office 365 includes what is called a "basic team site". This basic team site does not exist. "basic team site" is mentioned twice in the documentation.

The second location it is mentioned even gives a guide how to reach the site, see this parthttps://docs.microsoft.com/en-us/office365/admin/setup/set-up-file-storage-and-sharing?view=o365-worldwide#explore-your-team-site

Following those steps clearly shows that a new Office 365 Business Premium subscription does NOT contain the "basic team site".

This is highly confusing for a new user as the entire guide to Set up file storage and sharing is one of the basic guides one follows after getting a Business Premium license.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3c454f78-7336-f872-c945-23c2997856af
* Version Independent ID: 4d7e4a09-122e-d291-769b-93a4dffb0942
* Content: [Set up Office 365 file storage and sharing](https://docs.microsoft.com/en-us/office365/admin/setup/set-up-file-storage-and-sharing?view=o365-worldwide#explore-your-team-site)
* Content Source: [Office365-Admin/setup/set-up-file-storage-and-sharing.md](https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/blob/public/Office365-Admin/setup/set-up-file-storage-and-sharing.md)
* Service: **o365-administration**
* GitHub Login: @twernermsft
* Microsoft Alias: **twerner**

**Comment by thernstig** [comment]
I also noted that this site https://docs.microsoft.com/en-us/office365/admin/setup/cutomize-team-site?view=o365-worldwide says:

>  Office 365 subscriptions come with a classic team site. We recommend leaving it alone and creating a new team site that's connected to an Office 365 group.

That was simply not the case for me when signing up for Office 365 Business Premium. No such site exists as a SharePoint team site when I use SharePoint online.

**Comment by nenonix** [comment]
Thank you @thernstig. We appreciate your effort and feedback that helps us improve the content. We will post back here when we have amended the document in question.

**Comment by nenonix** [comment]
@thernstig I apologize for the delay in getting back to you. I have researched the issue thoroughly and it appears that you need to create a team to access what the documents call "a basic team site".  See https://support.office.com/article/create-a-team-in-teams for details. I hope that clears it up for you.

**Comment by thernstig** [comment]
@nenonix thank you for the investigation. I unfortunately do not think that is the solution to the question.

There are two problems with the answer:
1. The answer gives a solution of how I can create a team to get "a basic team site". However, the documentation in this issue points out that it should already exist. As such, the documentation is incorrect.
2. A "Microsoft Team" is the new "Slack-like" functionality in Office. Creating a team in "Microsoft Teams" is one way that will automatically create a team site. But other things will do this as well, such as creating a group in the admin panel. Are you thus absolutely sure the referral to the term "basic team site" really only refers to "Microsoft Teams". I suppose this question is really not so important, as question 1. is really where the heart of the problem lies.

**Comment by nenonix** [comment]
@thernstig Thank you for the clarity.  If I understand you correctly the content wording gives the impression of a default or basic site existing when what exists, by default, is the basic capacity to create a site but no site as such. And, if the problem here boils down to the term "basic team site" --how could it be re- worded or explained more clearly? What would you suggest?

**Comment by thernstig** [comment]
@nenonix I am not sure it is just the wording, but moreso that one does not exist. Let me explain how a new user (me) started out:

I created my first ever Office 365 Business Premium account 2 months ago. The first thing I did was to read the documentation here: https://docs.microsoft.com/en-us/office365/admin/setup/setup?view=o365-worldwide&tabs=BusPremium

I read all sub-pages under the main "Setup" page. One of these sub-pages is the one reported in this issue namely https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/blob/public/Office365-Admin/setup/set-up-file-storage-and-sharing.md or even more specifically this section: https://docs.microsoft.com/en-us/office365/admin/setup/set-up-file-storage-and-sharing?view=o365-worldwide#explore-your-team-site

If you read these guides from top-to-bottom and end up there, you do **not have a team site**. Nowhere in the guide before this has a team site been setup. So non exist. No matter if it is called a "team site" or "basic team site", it does not exist.

So it is really confusing if one tries to look for it.

**Comment by tseward** [comment]
@thernstig Any plan that includes SharePoint Online will come with https://tenatName.sharepoint.com which is a team site. Not all plans have SharePoint Online, but Business Premium does.

**Comment by thernstig** [comment]
@tseward I am not sure what you are saying is true (my memory escape me now). But for example in the link I pasted, the second paragraph reads:

> On the Office 365 home screen, choose SharePoint and then choose your team site link on the left-hand navigation menu.

This did not exist.

**Comment by nenonix** [comment]
@thernstig Thanks for your input, I can certainly see your point. I have reached out to the author of the document to step in and help us make this clearer.

**Comment by nenonix** [comment]
@thernstig thank you for your patience. Here is what I can tell you 
Microsoft did include a classic SharePoint site for new O365 customers but when the new/modern SharePoint sites came out, the classic site was hidden - so you are right, there is no site. This may change but I will remove that reference so that we don't confuse more people. Thank you for pointing this out, we really appreciate feedback that improves the content.

**Comment by thernstig** [comment]
@nenonix Thanks for getting to the bottom of this. Since I spent quite some time reading a lot of articles when I first got my Business Premium subscription and noticed the mention of "basic or classic site" it confused me quite a bit and had me loose some time trying to find where they were. So I am happy you found the reason why it was worded like that! Hopefully your findings and and revised version of documentation will save someone else some time :)
